scipy/optimize/_differentialevolution.py

```diff
@@ -697,8 +703,7 @@ def _mutate(self, candidate):
 
         fill_point = rng.randint(0, self.parameter_count)
 
-        if (self.strategy == 'randtobest1exp' or
-                self.strategy == 'randtobest1bin'):
+        if self.strategy == ['currenttobest1exp', 'currenttobest1bin']:
```

Comment:
No, this was supposed to be `if self.strategy in ['currenttobest1exp', 'currenttobest1bin']:`

Reply:
well that was silly, blame it on being in a rush